### DISCUSSION TITLE
OBS Studio (Install) package lags behind OBS Studio package by at least a week or more

### DISCUSSION THREAD

**Discussion by KattPhloxworthy** [chocolatey-community/chocolatey-packages#1842] [Q&A] [answered]
Why does the OBS Studio (Install) package lag by at least a week or more behind the OBS Studio package when an update to the latter drops?    I realize there were rapid-fire releases of OBS Studio recently owing to 27.2.x having issues with NVidia RTX-based cards, causing full-system crashes, but still, there is no reason to issue the package whose sole content is a dependency for the (Install) version of that package.

**Comment by gep13** [comment]
@KattPhloxworthy I am not sure that I follow exactly what you are saying here.

Can you provide links to the package, or package versions, that you are saying that has been impacted?

**Comment by pauby** [comment, marked as answer]
The obs packages are not maintained in this repository. Use the 'Contact Maintainers' link on the [package page](https://community.chocolatey.org/packages/obs-studio.install) to contact the maintainer directly.

**Comment by TheCakeIsNaOH** [comment]
See this issue for why the obs-studio.install package is sometimes out of date: https://github.com/AeliusSaionji/chocopkgs/issues/40
